newIDE/app/src/InstancesEditor/InstancesRenderer/LayerRenderer.js

```diff
@@ -533,7 +525,36 @@ export default class LayerRenderer {
     threePlaneTexture.magFilter = filter;
     threePlaneTexture.wrapS = THREE.ClampToEdgeWrapping;
     threePlaneTexture.wrapT = THREE.ClampToEdgeWrapping;
-    threePlaneMaterial.map = threePlaneTexture;
+
+    // Create the plane that will show this texture.
+    const threePlaneGeometry = new THREE.PlaneGeometry(1, 1);
+    this._threePlaneGeometry = threePlaneGeometry;
+    const noGammaCorrectionShader: THREE.ShaderMaterialParameters = {
```

Comment:
Add a comment to explain that this gamma correction is already done by Pixi so we don't want Three to do it again.